src/getCoinCombination.test.js

```diff
@@ -7,4 +7,52 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should return correct coin combination for 0 cents', () => {
```

Comment:
How do I find out what the "correct coin combination" means when I read your test without any requirements?